database/seeds/UsersTableSeeder.php

```diff
@@ -0,0 +1,23 @@
+<?php
+use Carbon\Carbon;
+use Illuminate\Database\Seeder;
+use Illuminate\Support\Facades\DB;
+
+class UsersTableSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        DB::table('users')->insert([
+        'name' =>         'test',
+        'email' =>        'dummy@email.com',
+        'password' =>     bcrypt('test1234'),
+        'created_at' =>  Carbon::now(),
+        'updated_at' =>   Carbon::now(),
+      ]);
```

Comment:
インデント崩れちゃってます
align合わせたいなら一番長いキーのところに `=>` を合わせるといいと思います
```suggestion
        DB::table('users')->insert([
            'name'       => 'test',
            'email'      => 'dummy@email.com',
            'password'   => bcrypt('test1234'),
            'created_at' => Carbon::now(),
            'updated_at' => Carbon::now(),
        ]);
```